sia-react-native/classes/user.js

```diff
@@ -1,4 +1,5 @@
 import statisticList from "./statisticList";
```

Comment:
File as a whole does not appear to be different from master (no added functionality, minor reshuffling of function locations, etc.). Consider pulling up to date user.js file from master before merge.